api.planx.uk/modules/file/file.test.ts

```diff
@@ -101,6 +106,26 @@ describe("File upload", () => {
         });
       expect(mockPutObject).toHaveBeenCalledTimes(1);
     });
+
+    it("should generate a correct URL on production", async () => {
+      vi.stubEnv("API_URL_EXT", "https://api.editor.planx.dev");
```

Comment:
nit: based on test description, I'd expect this to be `.uk` ! 

Also perhaps worth updating `expect(res.body)` > `fileUrl `to include base URL prefix stubbed here - else we're only actually testing params behavior?

Reply:
Good catch - some heavy handed copy/paste here clearly 😅 

Test updated here - [8f1ff37](https://github.com/theopensystemslab/planx-new/pull/3989/commits/8f1ff37873dfcc3e8007c1d9478c3bb0fdc956cf)